draft-js-plugins-editor/src/Editor/index.js

```diff
@@ -125,6 +112,35 @@ class PluginEditor extends Component {
     getEditorRef: this.getEditorRef,
   });
 
+  getEditorStateWithCombinedDecorators() {
+    const decorators = this.resolveDecorators();
+    const compositeDecorator = createCompositeDecorator(
+      decorators.filter((decorator) => !this.decoratorIsCustom(decorator)),
+      this.getEditorState,
+      this.onChange,
+      this.props.CompositeDraftDecorator);
+
+    const customDecorators = decorators
+      .filter((decorator) => this.decoratorIsCustom(decorator));
+
+    const multiDecorator = new MultiDecorator(
+      [
+        ...customDecorators,
+        compositeDecorator,
+      ]
+    );
+
```

Comment:
Following the same reasoning as in https://github.com/emily-plummer/draft-js-plugins/pull/1/commits/bd2ab391324c97ce4a09e0a2528cf2c4f86148d7, I'd replace this:

```js
    const multiDecorator = new MultiDecorator(
      [
        ...customDecorators,
        compositeDecorator,
      ]
    );
```

with this:

```js
    const multiDecorator = new MultiDecorator(
      customDecorators.push(compositeDecorator));
```

So that we're as close to the updated branch as possible